ambry-protocol/src/main/java/com.github.ambry.protocol/CatchupStatusAdminRequest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.protocol;
+
+import java.io.DataInputStream;
+import java.io.IOException;
+
+
+/**
+ *  An admin request used to determine if the peers of a storage node have caught up to it.
+ */
+public class CatchupStatusAdminRequest extends AdminRequest {
+  private static final short VERSION_V1 = 1;
```

Comment:
minor: the V is redundant since the name already includes "VERSION." Seems like we do this in other places though.

Reply:
Yeah, was just being consistent with the other request classes.